custom_model_runner/datarobot_drum/drum/gpu_predictors/nemo_predictor.py

```diff
@@ -0,0 +1,239 @@
+"""
+Copyright 2024 DataRobot, Inc. and its affiliates.
+All rights reserved.
+This is proprietary source code of DataRobot, Inc. and its affiliates.
+Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+import csv
+import io
+import logging
+import os
+import signal
+import subprocess
+import atexit
+
+import numpy as np
+from openai import OpenAI
+from threading import Thread
+from datarobot_drum import RuntimeParameters
+from datarobot_drum.drum.adapters.model_adapters.python_model_adapter import RawPredictResponse
+from datarobot_drum.drum.common import SupportedPayloadFormats
+from datarobot_drum.drum.enum import (
+    LOGGER_NAME_PREFIX,
+    PayloadFormat,
+    StructuredDtoKeys,
+)
+from datarobot_drum.drum.exceptions import DrumCommonException
+from datarobot_drum.drum.language_predictors.base_language_predictor import BaseLanguagePredictor
+from datarobot_drum.resource.drum_server_utils import wait_for_server, DrumServerProcess
+
+RUNNING_LANG_MSG = "Running environment: Nemo Inference Microservices."
```

Comment:
```suggestion
RUNNING_LANG_MSG = "Running environment: NeMo Inferencing Microservice."
```
This appears to be the official name? https://developer.nvidia.com/blog/nvidia-nim-offers-optimized-inference-microservices-for-deploying-ai-models-at-scale/